### ISSUE TITLE
RFE: connect to a weather API

### ISSUE THREAD

**Issue by LutzLange** [jamesread/dotnetcore-city-info#3]
It would be a good extension to have your info app connect to a real weather api and only show static data as a fall back.

**Comment by jamesread** [comment]
You're right, this would be really cool. We need to find a free and simple weather API that works based on cities. 

http://samples.openweathermap.org/data/2.5/forecast?id=524901&appid=b6907d289e10d714a6e88b30761fae22 looks like a good example (Moscow) in t hat it seems simple to use. Will give this some additional thought.
